.gitignore

```diff
@@ -24,3 +24,8 @@ current_counterexample.eqc
 *.log
 *.log.*
 doc
+# Emacs Backup files
+*~
+# Emacs temporary files
+.#*
```

Comment:
Hi, thanks for this. Shouldn't the `#` be escaped?

Reply:
As far as I know it needs to be deleted if it is the first character.  
That made me think that one is missing:   \#*